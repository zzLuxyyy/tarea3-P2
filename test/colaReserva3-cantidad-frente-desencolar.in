crearColaReservas
cantidadColaReservas
crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1966
crearLibro 411222335 "Todos los fuegos el fuego" "Julio" "Cortázar" "Ocho cuentos que combinan lo cotidiano con lo fantástico" 1
crearReserva
encolarColaReservas    
cantidadColaReservas
desencolarColaReservas
crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1961
crearLibro 421233446 "La ratesa" "Günter" "Grass" "Una alegoría postnuclear narrada por una rata parlante que reflexiona sobre la historia" 2
crearReserva
encolarColaReservas
cantidadColaReservas
frenteColaReservas
desencolarColaReservas
liberarColaReservas
crearColaReservas
crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1968
crearLibro 431244557 "Lo bello y lo triste" "Yasunari" "Kawabata" "Un escritor se reencuentra con una amante del pasado en una historia de dolor y deseo" 1
crearReserva
encolarColaReservas
frenteColaReservas
crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 05/06/1967
crearLibro 843760494 "Cien años de soledad" "Gabriel" "García Márquez" "Es la historia de la familia Buendía a lo largo de siete generaciones en el pueblo ficticio de Macondo" 3
crearReserva
encolarColaReservas
frenteColaReservas
crearSocio 23456789 Lucia Perez 10 03 2010
crearFecha 01/01/1961
crearLibro 421233446 "La ratesa" "Günter" "Grass" "Una alegoría postnuclear narrada por una rata parlante que reflexiona sobre la historia" 2
crearReserva
encolarColaReservas
frenteColaReservas
cantidadColaReservas
crearSocio 45678901 Juan Diaz 10 03 1999
crearFecha 05/06/1967
crearLibro 843760494 "Cien años de soledad" "Gabriel" "García Márquez" "Es la historia de la familia Buendía a lo largo de siete generaciones en el pueblo ficticio de Macondo" 3
crearReserva
encolarColaReservas
cantidadColaReservas
desencolarColaReservas
frenteColaReservas
cantidadColaReservas
liberarColaReservas
Fin