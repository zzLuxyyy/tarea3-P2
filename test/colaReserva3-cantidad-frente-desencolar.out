1>Se ejecutó 'crearTColaReservas' exitosamente
2>La cantidad de reservas en la cola es 0
3>Socio de ci 12345678 creado exitosamente.
4>5>6>Reserva de libro de isbn 411222335 a socio 12345678 creada exitosamente.
7>Se ejecutó 'encolarTColaReservas' exitosamente.
8>La cantidad de reservas en la cola es 1
9>Se ejecutó 'desencolarTColaReservas' exitosamente
10>Socio de ci 12345678 creado exitosamente.
11>12>13>Reserva de libro de isbn 421233446 a socio 12345678 creada exitosamente.
14>Se ejecutó 'encolarTColaReservas' exitosamente.
15>La cantidad de reservas en la cola es 1
16>Reserva obtenida al ejecutar frente:
Reserva de libro La ratesa por Jim Dear.
17>Se ejecutó 'desencolarTColaReservas' exitosamente
18>Se ejecutó 'liberarTColaReservas' exitosamente
19>Se ejecutó 'crearTColaReservas' exitosamente
20>Socio de ci 12345678 creado exitosamente.
21>22>23>Reserva de libro de isbn 431244557 a socio 12345678 creada exitosamente.
24>Se ejecutó 'encolarTColaReservas' exitosamente.
25>Reserva obtenida al ejecutar frente:
Reserva de libro Lo bello y lo triste por Jim Dear.
26>Socio de ci 12345678 creado exitosamente.
27>28>29>Reserva de libro de isbn 843760494 a socio 12345678 creada exitosamente.
30>Se ejecutó 'encolarTColaReservas' exitosamente.
31>Reserva obtenida al ejecutar frente:
Reserva de libro Lo bello y lo triste por Jim Dear.
32>Socio de ci 23456789 creado exitosamente.
33>34>35>Reserva de libro de isbn 421233446 a socio 23456789 creada exitosamente.
36>Se ejecutó 'encolarTColaReservas' exitosamente.
37>Reserva obtenida al ejecutar frente:
Reserva de libro Lo bello y lo triste por Jim Dear.
38>La cantidad de reservas en la cola es 3
39>Socio de ci 45678901 creado exitosamente.
40>41>42>Reserva de libro de isbn 843760494 a socio 45678901 creada exitosamente.
43>Se ejecutó 'encolarTColaReservas' exitosamente.
44>La cantidad de reservas en la cola es 4
45>Se ejecutó 'desencolarTColaReservas' exitosamente
46>Reserva obtenida al ejecutar frente:
Reserva de libro Cien años de soledad por Jim Dear.
47>La cantidad de reservas en la cola es 3
48>Se ejecutó 'liberarTColaReservas' exitosamente
49>Fin.
